### PR TITLE
Discourse: default to update mode 2

### PR DESCRIPTION
**Pull request by Beanow**
Flips the switch, so the new update mode (#1464) is used.

Test plan: `node bin/sourcecred.js discourse https://sourcecred-test.discourse.group`
On an empty cache, manual runs such as these should load correctly and have 100% coverage of all public content. An easy way to spot if mode 2 is being used, is the lack of a `discourse/posts` step in the CLI.

### CODE REVIEW

**Comment by Beanow** [comment]
Changed this to use the enum instead of number.
Now running another extended manual test given there were a few improvements in the previous PRs.

**Comment by Beanow** [comment]
Tested with a few real examples:
```
node bin/sourcecred.js discourse https://sourcecred-test.discourse.group
node bin/sourcecred.js discourse https://discourse.sourcecred.io --weights ...
node bin/sourcecred.js discourse https://discourse.sustainoss.org
node bin/sourcecred.js discourse https://forum.osc.dial.community
```

Using these https://github.com/sourcecred/cred/blob/master/weights.json for the sourcecred forum.

<details><summary>Empty cache run:</summary>

```
  GO   load-sourcecred-test.discourse.group
  GO   discourse
  GO   discourse/topics
 DONE  discourse/topics: 51s
  GO   discourse/likes
 DONE  discourse/likes: 5s
 DONE  discourse: 56s
  GO   compute-cred
 DONE  compute-cred: 68ms
 DONE  load-sourcecred-test.discourse.group: 56s
  GO   load-discourse.sourcecred.io
  GO   discourse
  GO   discourse/topics
 DONE  discourse/topics: 3m 16s
  GO   discourse/likes
 DONE  discourse/likes: 2m 23s
 DONE  discourse: 5m 39s
  GO   compute-cred
 DONE  compute-cred: 1s
 DONE  load-discourse.sourcecred.io: 5m 40s
  GO   load-discourse.sustainoss.org
  GO   discourse
  GO   discourse/topics
 DONE  discourse/topics: 793ms
  GO   discourse/likes
 DONE  discourse/likes: 41s
 DONE  discourse: 42s
  GO   compute-cred
 DONE  compute-cred: 353ms
 DONE  load-discourse.sustainoss.org: 42s
  GO   load-forum.osc.dial.community
  GO   discourse
  GO   discourse/topics
 DONE  discourse/topics: 2m 14s
  GO   discourse/likes
 DONE  discourse/likes: 53s
 DONE  discourse: 3m 7s
  GO   compute-cred
 DONE  compute-cred: 400ms
 DONE  load-forum.osc.dial.community: 3m 8s
```

</details>

<details><summary>Second run:</summary>

```
  GO   load-sourcecred-test.discourse.group
  GO   discourse
  GO   discourse/topics
 DONE  discourse/topics: 711ms
  GO   discourse/likes
 DONE  discourse/likes: 4s
 DONE  discourse: 5s
  GO   compute-cred
 DONE  compute-cred: 101ms
 DONE  load-sourcecred-test.discourse.group: 5s
  GO   load-discourse.sourcecred.io
  GO   discourse
  GO   discourse/topics
 DONE  discourse/topics: 741ms
  GO   discourse/likes
 DONE  discourse/likes: 58s
 DONE  discourse: 58s
  GO   compute-cred
 DONE  compute-cred: 1s
 DONE  load-discourse.sourcecred.io: 1m 0s
  GO   load-discourse.sustainoss.org
  GO   discourse
  GO   discourse/topics
 DONE  discourse/topics: 826ms
  GO   discourse/likes
 DONE  discourse/likes: 41s
 DONE  discourse: 42s
  GO   compute-cred
 DONE  compute-cred: 583ms
 DONE  load-discourse.sustainoss.org: 43s
  GO   load-forum.osc.dial.community
  GO   discourse
  GO   discourse/topics
 DONE  discourse/topics: 718ms
  GO   discourse/likes
 DONE  discourse/likes: 50s
 DONE  discourse: 51s
  GO   compute-cred
 DONE  compute-cred: 535ms
 DONE  load-forum.osc.dial.community: 51s
```

</details>

TL;DR they look good :+1:
